event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/feed/common/Paging.java

```diff
@@ -4,10 +4,18 @@
 
     private final String previous;
     private final String next;
+    private final String head;
+    private final String last;
 
-    public Paging(final String previous, final String next) {
```

Comment:
As mentioned above, perhaps this could just be a Map of Link to String (URI)

Reply:
Not doing

Reply:
not changing anything